utils/HACKING.adoc

```diff
@@ -52,4 +52,16 @@ tools to test the new release, and if you update *after* that you risk
 breaking them again without them noticing.
 
 For example, the magic numbers for 4.13 were updated in
-  dd7927e156b7cb2f9
+  https://github.com/ocaml/ocaml/commit/dd7927e156b7cb2f9cb73d2d54a15a9c8192139\2[dd7927e]
```

Comment:
I think it is better to remove the link to this commit, since the commit will point to files that are no longer versioned.